zen_queries/__init__.py

```diff
@@ -9,3 +9,13 @@
 
 
 __version__ = "2.1.0"
+
+__all__ = [
+    "fetch",
+    "queries_dangerously_enabled",
+    "queries_disabled",
+    "QueriesDisabledError",
+    "render",
+    "SimpleTemplateResponse",
+    "TemplateResponse",
+]
```

Comment:
The explicit reexport of the public API is required for the library to work with an app that runs mypy with the `implicit_reexport = false` setting. It also makes linters like flake8 a lot more happy.